### PR TITLE
Peng/600 ibc sending design

### LINKED ISSUE #600: IBC design

### PR DESCRIPTION
**Pull request by nylira**
### Issue
Closes #600

### Screenshots
Unfortunately I can't take a screenshot, and I'm also not sure if this PR results in a working `<select` because I don't have any tokens to send. I'll be back tonight to look at it. @mappum, if you need it sooner please help me look at it.

### CODE REVIEW

**Comment by nylira** [comment]
Now merged with `develop` and uses Prettier formatting. CircleCI passes!

**Comment by nylira** [comment]
Yay, the coverage meter is passing! This is ready for review and merge.

File: app/src/renderer/vuex/modules/wallet.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ export default ({ commit, node }) => {
     key: { address: "" },
     history: [],
     historyLoading: false,
-    denoms: []
+    denoms: [],
+    zoneIds: ["cosmos-hub-1", "cosmos-hub-2"]
```
Comment (faboweb) [inline app/src/renderer/vuex/modules/wallet.js]: Should we maybe put this in the config file? Can be done later.

Comment (nylira) [reply app/src/renderer/vuex/modules/wallet.js]: Yes, for sure. I wasn't clear on how the backend would be implemented so I left it in `wallet.js`

**Comment by jolesbi** [comment]
why no screenshots?

**Comment by nylira** [comment]
@jolesbi I can't view the Send page without any tokens. I'll create a temporary link to try for a screenshot.

**Comment by jolesbi** [comment]
fabo should be able to get you some tokens... no?

**Comment by nylira** [comment]
### Screenshots

screen shot 2018-04-05 at 9 44 03 pm
screen shot 2018-04-05 at 9 43 51 pm

**Comment by nylira** [comment]
@jolesbi pulling chain id from the address is a "for later" feature. Cosmos SDK doesn't support it yet and since we're running two testnets from the same 0.13.1, the address formats will be indistinguishable from each other.

Regarding coins, I don't think I'm running on the right testnet yet to accept any coins. I'm just running a local network right now.

**Comment by jolesbi** [comment]
this looks great. thanks for the info peng!

**Pull request merged**